added form upload sertifikat in laporan page

bayun2445 opened the pull request.

bayun2445 committed:
added form upload sertifikat in laporan page

Chillrend commented:
- Cek dokumen laporan tidak terupload
<img width="736" alt="image" src="https://github.com/Chillrend/SIMBKM/assets/38966867/86e734c1-1af9-4b31-80f4-86f1adfa2cf0">

- Cek di role dosen, belum ada info sertifikat dan tombol untuk melihat file sertifikat. buat juga tombol utk melihat file laporan.

bayun2445 committed:
Merge branch 'Chillrend:master' into master

bayun2445 committed:
updated documents upload form in laporan page

merged updateSertifikat() to update() in Laporan Controller
merged the upload file forms into one form

bayun2445 committed:
Merge branch 'update/laporan'

bayun2445 committed:
Merge branch 'Chillrend:master' into master

lanang0809-pro committed:
Update view detail-laporan Dosbing

lanang0809-pro committed:
Merge branch 'master' of https://github.com/Lanang0809/SIMBKM

bayun2445 committed:
Merge pull request #1 from Lanang0809/master

Update view detail-laporan Dosbing

bayun2445 commented:
updated form upload sertifikat
added sertifikat view document in dosbing laporan page

lanang0809-pro committed:
Add some new HTML elements

Menambahkan form input file  pada halaman create-logbook admin
Menambahkan button view dokumen pada halaman detail-logbook Dosbing dan detail-logbook KPS.

Lanang0809 committed:
Merge branch 'bayun2445:master' into master

bayun2445 committed:
Merge pull request #2 from Lanang0809/master

Add some new HTML elements

bayun2445 committed:
added post dokumen logbook field

lanang0809-pro committed:
Menanmpilkan file sertifikat dan Pembaruan Tampilan detail-laporan dosbing

Membuat fungsi untuk menampilkan file sertifikat di DosbingController,
Membuat rute ke halaman untuk menampilkan file sertifikat,
Memperbarui struktur elemen HTML pada halaman detail-laporan dosbing.

Lanang0809 committed:
Merge branch 'bayun2445:master' into master

bayun2445 committed:
Merge pull request #3 from Lanang0809/master

Memperbarui tampilan detail-laporan dosbing dan Menampilkan file sertifikat

bayun2445 committed:
Added logbook document upload form

bayun2445 committed:
Merge branch 'master' of https://github.com/bayun2445/SIMBKM

bayun2445 committed:
Update view logbook document link

lanang0809-pro committed:
Add Edit and Delete in kelola-akun

Lanang0809 committed:
Merge branch 'bayun2445:master' into master

bayun2445 committed:
Added edit, update, remove function

bayun2445 committed:
Merge pull request #4 from Lanang0809/master

Add Edit and Delete in kelola-akun

lanang0809-pro committed:
Added edit-akun view

lanang0809-pro committed:
Merge branch 'master' of https://github.com/Lanang0809/SIMBKM

Lanang0809 committed:
Merge branch 'bayun2445:master' into master

bayun2445 committed:
Merge pull request #5 from Lanang0809/master

Added user-akun view
